### PR TITLE
Status: 2023q3: mfsBSD: corrections, suggestions

### PR DESCRIPTION
**Pull request by grahamperrin**
Fix markup for hard line breaks.

https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#paragraphs

Matuška (not Matuska).

For a named person – with no visible hint of a `mailto:` URL – reference to their home page may be less surprising than launch of an email application.

Whilst here: other minor changes (suggestions).

----

As this relates to a [status report](https://github.com/freebsd/freebsd-doc/pulls?q=is%3Apr+label%3A%22status+report%22), please treat it as an exception to the rule that excludes my PRs from this repository. Thank you.

### CODE REVIEW

**Comment by grahamperrin** [comment]
@soobinrho and reviewers

Re: https://github.com/freebsd/freebsd-doc/pull/252#discussion_r1319536775 I am to blame for the incorrect markup. Sorry. 

I realised the mistake at my <http://localhost:1313/status/report-2023-07-2023-09/mfsbsd/> view of d5f71f6b45b142e66b193cd64947bf0f3ea48377, here's a PDF: 

* [d5f71f6b45b142e66b193cd64947bf0f3ea48377.pdf](https://github.com/freebsd/freebsd-doc/files/12642605/d5f71f6b45b142e66b193cd64947bf0f3ea48377.pdf)

Corrected, and with suggested changes: 

* [Integrate mfsBSD into the Release Building Tools.pdf](https://github.com/freebsd/freebsd-doc/files/12642608/Integrate.mfsBSD.into.the.Release.Building.Tools.pdf)

I am uncertain about some suggestions, so do (please) comment if there is wrongness.

### Home page for Martin Matuška

The 2009 white paper gave the mfsBSD home page as <http://people.freebsd.org/~mm/mfsbsd/>, which redirects to <https://mfsbsd.vx.sk/> where @mmatuska offers <http://blog.vx.sk/?user_language=en> for his blog. <https://github.com/mmatuska/> more concisely refers to: 

* `http://blog.vx.sk/`

File: website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc [hunk 1]
```diff
@@ -12,19 +12,22 @@ Contact: Soobin Rho <soobinrho@FreeBSD.org>
...
+Upstream mfsBSD is maintained in the link:https://github.com/mmatuska/mfsbsd[repository on GitHub].
 
 ==== Purpose of this Project
 
-This project integrates mfsBSD into the FreeBSD release tool set, creating an additional target of mfsBSD images ([.filename]#.img# and [.filename]#.iso# files) into the [.filename]#/usr/src/release/Makefile#.
-Until now, mfsBSD only existed outside the FreeBSD release tool chain, and only the -release versions have been produced.
-With this project, mfsBSD images will be available at the official FreeBSD release page, and you will also be able to build mfsBSD yourself by invoking `cd /usr/src/release && make release WITH_MFSBSD=1`, which will then create [.filename]#mfsbsd-se.img# and [.filename]#mfsbsd-se.iso# at [.filename]#/usr/obj/usr/src/${ARCH}/release/#.
+This project integrates mfsBSD into the FreeBSD release tool set, creating an additional target of mfsBSD images ([.filename]#.img# and [.filename]#.iso# files) in [.filename]#/usr/src/release/Makefile#.
+Prior to integration, mfsBSD only existed outside the FreeBSD release tool chain, and only -RELEASE versions were produced.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc:22]: I changed _-release_ to uppercase _-RELEASE_. 

Is this appropriate?

**Review by soobinrho — APPROVED** [review]
Awesome 👍👍

File: website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc (hunk 1 above)
Comment (soobinrho) [reply website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc:22]: That will be a good change!

**Comment by grahamperrin** [comment]
@soobinrho, thank you. 

----

Reviewers, please excuse the force-push. I know, [it is (or was) discouraged](https://github.com/freebsd/freebsd-quarterly/commit/06726494b8083494ff9c2b68719504d6493a55dd#diff-1194561cbea346acb1bea37d81fdf5c70def982dcbb0c64eca2a96bc51941124L24). My force in cases such as this is to add pull request, closure, and sign-off lines for `.patch` file purposes. 

Here: <https://github.com/freebsd/freebsd-doc/pull/257.patch>

File: website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc [hunk 2]
```diff
@@ -12,19 +12,22 @@ Contact: Soobin Rho <soobinrho@FreeBSD.org>
 "mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, USB storage device, or optical medium.
 It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
 
-mailto:mm@FreeBSD.org[Martin Matuska] is the creator of mfsBSD, and he is also the author of the original link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper], which is where the excerpt above was taken from.
-The upstream mfsBSD is maintained at the link:https://github.com/mmatuska/mfsbsd[GitHub mfsBSD repository].
+http://blog.vx.sk/[Martin Matuška] is the creator of mfsBSD.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc:15]: We usually link to e-mail addresses on names in status reports. Moreover, the linked blog has many references to different things that are not about mfsBSD, nor even about FreeBSD.

Please put the e-mail address back.

**Review by lsalvadore — APPROVED** [review]
Approved once the e-mail address is restored.

File: website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc (hunk 2 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc]: Thanks, resolved by [`aa9fcb7` (#257)](https://github.com/freebsd/freebsd-doc/pull/257/commits/aa9fcb7e81e9f0fc970bb6b536eba6017712a63c). 

Would you like the address for `mm@` to be visible? (As it is for `soobinrho@` at line 8.)

Comment (lsalvadore) [reply website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc]: No, thanks, no need to make it visible.

**Comment by lsalvadore** [comment]
Merged, thanks.

File: website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc (hunk 2 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-07-2023-09/mfsbsd.adoc]: All clear. I'll probably remember all of this for future mfsBSD reports.

**Pull request closed without merging**